src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertAdditionalServices.sql

```diff
@@ -17,19 +17,19 @@ BEGIN
 
```

Comment:
Can you remove the `USE` statement from the start of the file? The DB name won't be 'buyingcatalogue' in a dynamic environment. (Sorry, just noticed it after you removed the tabs.)